examples/basic-express-typescript/lib/route.ts

```diff
@@ -0,0 +1,27 @@
+
+import * as express from 'express'
+import * as parsers from 'body-parser'
+import * as cors from 'cors'
+import * as sync from '../../../fh-sync'
+
+const router = express.Router()
+
+// Mobile clients typically require CORS headers to be set
+router.use(cors())
+
+// Need to parse incoming JSON bodies
+router.use(parsers.json())
+
+// All sync requests are performed using a HTTP POST
+router.post('/:datasetId', (req: express.Request, res: express.Response, next: express.NextFunction) => {
+  // Invoke action in sync for specific dataset
+  sync.invoke(req.params.datasetId, req.body, function(err: any, result: any) {
```

Comment:
@wtrocki Any reason you changed the types of the callback parameters to `any`? The types seem to get inferred correctly, and in fact, `any` is a wider type than `err` is, which is `string|Error|null` in the typings.

Reply:
I did not compile for me - were complaining about missing types :)

Reply:
I think this might be a `noImplicitAny` issue? In any case, does it mean we need better typings for `result` here? It should always be `Object` right?